alerter/alerter.java

```diff
@@ -1,10 +1,14 @@
 public class Alerter {
     static int alertFailureCount = 0;
+    static int threshHold = 200;
     static int networkAlertStub(float celcius) {
         System.out.println("ALERT: Temperature is " + celcius + " celcius");
         // Return 200 for ok
         // Return 500 for not-ok
         // stub always succeeds and returns 200
+        if(celcius>threshHold){
+            return 500;
+        }
         return 200;
     }
     static void alertInCelcius(float farenheit) {

```

Comment:
The production code should not be aware of test code or test stub. It should be abstracted by the use of interface or function pointer.